src/lib.rs

```diff
@@ -65,7 +76,8 @@
 extern crate difference;
 #[macro_use] extern crate error_chain;
 
-use std::process::Command;
+use std::process::{Command, Output};
+use std::fmt;
```

Comment:
_When_ I think about it, I do it like

1. things from std
2. things from external crates
3. my things

and sort the individual blocks alphanumerically